apps/studio/src/components/PageEditor/FileAttachment.tsx

```diff
@@ -82,7 +79,7 @@ export const FileAttachment = ({
         />
       </Skeleton>
       <Text textStyle="body-2" textColor="base.content.medium" pt="0.5rem">
-        {`Maximum file size: ${MAX_FILE_SIZE_BYTES / 1000000} MB`}
+        {`Maximum file size: ${maxSizeInBytes / 1000000} MB`}
```

Comment:
nit: can just make the 1000000 a constant `ONE_MB_IN_BYTES` since we have a couple of areas of this use

Reply:
i did this but after searching through the codebase, i think this is the only use!